components/gfx/text/glyph.rs

```diff
@@ -782,17 +784,23 @@ impl<'a> Iterator for GlyphIterator<'a> {
             self.next_glyph_range()
```

Comment:
nit: While you're here, could you change this into an early return? Then you can de-indent all your code below.